tests/e2e/performance/test_pod_reattachtime.py

```diff
@@ -0,0 +1,136 @@
+import logging
+import pytest
+import ocs_ci.ocs.exceptions as ex
+import urllib.request
+import time
+from ocs_ci.framework.testlib import (
+    performance, E2ETest
+)
+from tests import helpers
+from ocs_ci.ocs import constants
+from ocs_ci.ocs.ocp import OCP
+
+log = logging.getLogger(__name__)
+
+
+@pytest.mark.polarion_id("OCS-2208")
+@performance
+class TestPVCCreationPerformance(E2ETest):
+    """
+    Test to verify PVC creation performance
+    """
+
+    @pytest.fixture()
+    def base_setup(
+        self, request, interface_iterate, storageclass_factory
+    ):
+        """
+        A setup phase for the test
+        Args:
+            interface_iterate: A fixture to iterate over ceph interfaces
+            storageclass_factory: A fixture to create everything needed for a
+                storageclass
+        """
+        self.interface = interface_iterate
+        self.sc_obj = storageclass_factory(self.interface)
+
+    @pytest.mark.usefixtures(base_setup.__name__)
+    def test_pvc_reattach_time_performance(self, pvc_factory, teardown_factory):
+        """
+        Test assign nodeName to a pod using RWX pvc
+        Performance in test_multiple_pvc_creation_measurement_performance
+        Each kernel (unzipped) is 892M and 61694 files
+        """
+        kernel_url = 'https://cdn.kernel.org/pub/linux/kernel/v4.x/linux-4.19.5.tar.gz'
```

Comment:
Is there a reason to specifically download the kernel files each time, instead of generating equivalent IO locally?
I think it can shorten the runtime by a lot, and will also save a lot of bandwidth down the road, while also delivering more consistent results across different machines and network connections.

Reply:
> Is there a reason to specifically download the kernel files each time, instead of generating equivalent IO locally?
> I think it can shorten the runtime by a lot, and will also save a lot of bandwidth down the road, while also delivering more consistent results across different machines and network connections.

The intention is to generate a lot of small files and directories which simulates a typical small file workload. Untarring the kernel bundle could generate ~60k files and we could simply make use of the tarball to generate 60k files each time we untar. 
We could work on such an implementation, but to start with this is the quickest way to simulate the workload. 

> 
> I also think it might be a good idea to move this over to `constants`

Sure, I'll move this to constants.

Reply:
since its used within this one test and scope of using it is low in other test, it might be fine, if we decide we can move it in subsequent pr.